keps/sig-scheduling/2926-job-mutable-scheduling-directives/README.md

```diff
@@ -534,7 +534,7 @@ kube-apiserver will accept node affinity/tolerations updates of jobs.
 
 ###### Are there any tests for feature enablement/disablement?
 
-No, unit and integration tests will be added prior to Beta graduation.
+Tests are in place.
```

Comment:
> Integration tests for enabling/disabling the feature are here: https://github.com/kubernetes/kubernetes/blob/457341c3d408097025af5a9b6f5917439c0debdd/test/integration/job/job_test.go#L1397

Can you link that to the KEP?

Also - the tests test when the feature is enabled and when the feature is disabled, not necessary the process of enablement disablement.

This is fine in this particular case, because the actual change is "in-memory" (the validation change), but might be worth making mentioning this explicitly.

Reply:
done.